exdb/models.py

```diff
@@ -136,6 +136,42 @@ def get_url(self, user):
             return reverse('view_experience', args=[self.pk])
         return reverse('edit', args=[self.pk])
 
+    def get_csv_row(self):
```

Comment:
You could change this to `convert_to_dict()` and return a dict for use with [csv.DictWriter](https://docs.python.org/3/library/csv.html#csv.DictWriter) in the view.